src/app/content/components/BookStyles.tsx

```diff
@@ -8,4 +8,5 @@ const Hoc = ({children, className}: {className?: string, children: (className?:
 
 export default styled(Hoc)`
   ${bookStyles}
+  // Insert custom styles into ContentPane
```

Comment:
not sure if this will be correctly ignored inside the css, might want to move it up to line 8